salon_creator/lib/screens/salon_creation_screen.dart

```diff
@@ -8,20 +20,529 @@ class SalonCreationScreen extends StatefulWidget {
+          );
+        },
+      ),
+    );
+  }
+
+  Widget _buildMediaPaneIcon(int i) {
+    return Stack(
+      alignment: Alignment.topRight,
+      clipBehavior: Clip.antiAliasWithSaveLayer,
+      children: [
+        Image(
+          fit: BoxFit.cover,
+          width: 80,
+          height: 45,
+          image: FileImage(
+            File(
+              _imageFiles[i].path,
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildMediaPaneAddButton(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        color: primaryColor,
+        border: Border.all(
+          color: primaryColor,
+          width: 2,
+        ),
+      ),
+      child: IconButton(
+        alignment: Alignment.center,
+        icon: Icon(
+          Icons.add_circle,
+          size: 28,
+          color: Colors.white,
+        ),
+        onPressed: () {
+          showBottomSheet(context);
+        },
+      ),
+    );
+  }
+
+  Widget _buildMediaPreviewer(
+      double screenHeight, double screenWidth, BuildContext context) {
+    return Container(
+      color: Color.fromRGBO(195, 195, 195, 0.3),
+      height: screenWidth * 0.563,
+      width: screenWidth,
+      child: _imageFiles.isEmpty
+          ? TextButton(
+              child: Icon(
+                Icons.add_circle,
+                size: 64,
+                color: primaryColor,
+              ),
+              onPressed: () {
+                showBottomSheet(context);
+              },
+            )
+          : Stack(
+              alignment: Alignment.topLeft,
+              children: [
+                Image(
+                  height: screenWidth * 0.563,
+                  width: screenWidth,
+                  fit: BoxFit.cover,
+                  image: FileImage(
+                    File(
+                      _imageFiles[index].path,
+                    ),
+                  ),
+                ),
+                _buildDeleteButton(context),
+              ],
+            ),
+    );
+  }
+
+  Widget _buildDeleteButton(BuildContext context) {
+    return Container(
+      width: 40,
+      height: 40,
+      margin: EdgeInsets.all(4),
+      decoration: BoxDecoration(
+        color: primaryColor,
+        shape: BoxShape.circle,
+      ),
+      child: IconButton(
+        onPressed: () {
+          _buildDeleteConfirmationDialog(context);
+        },
+        icon: Icon(
+          Icons.close,
+          color: Colors.white,
+        ),
+      ),
+    );
+  }
+
+  void _buildDeleteConfirmationDialog(BuildContext context) {
+    return showCustomDialog(
+      context: context,
+      title: Text(
+        "本当に削除しますか?",
+        textAlign: TextAlign.center,
+        style: TextStyle(
+          fontSize: 18,
+        ),
+      ),
+      content: "選択されている画像・動画が削除されます",
+      leftButtonText: "削除する",
+      leftFunction: () {
+        setState(() {
+          _imageFiles.removeAt(index);
+          index -= 1;
```

Comment:
```suggestion
          selectedImageIndex -= 1;
```
rename variable for better readability?